scripts/generators/generator.py

```diff
@@ -0,0 +1,28 @@
+import random
+import numpy as np
+import pandas as pd
+from .csv_writer import CSVWriter
+from abc import ABC, abstractmethod
+
+class Generator(ABC):
+
+    def __init__(self, seed: int = None) -> None:
+        self.csvw = CSVWriter()
+        if not seed == None:
+            random.seed(seed)
+
+    @abstractmethod
+    def generate(self, quantity: int = 1) -> None:
```

Comment:
```suggestion
    def generate(self, quantity: int) -> None:
```
I think in `@abstarctmethod` shouldn't be declared value.

Reply:
This makes implementation easier as the editor automatically suggests a default value in subclasses. I did not come across any contraindications.